README.md

```diff
@@ -1,424 +1,159 @@
-- [google-cloud-logging API documentation](https://googleapis.dev/ruby/google-cloud-logging/latest)
-- [google-cloud-logging on RubyGems](https://rubygems.org/gems/google-cloud-logging)
-- [Google Cloud Logging documentation](https://cloud.google.com/logging/docs/)
-
-### Cloud Monitoring
-
-- `gem install google-cloud-monitoring`
-- [google-cloud-monitoring README](google-cloud-monitoring/README.md)
-- [google-cloud-monitoring API documentation](https://googleapis.dev/ruby/google-cloud-monitoring/latest)
-- [google-cloud-monitoring on RubyGems](https://rubygems.org/gems/google-cloud-monitoring)
-- [Google Cloud Monitoring documentation](https://cloud.google.com/monitoring/docs)
+Idiomatic Ruby client libraries for
+[Google Cloud Platform](https://cloud.google.com/) APIs.
 
-### Cloud OS Login
+* [Homepage](https://github.com/googleapis/google-cloud-ruby/)
+* [Reference documentation](https://cloud.google.com/ruby/docs/reference)
 
-- `gem install google-cloud-os_login`
-- [google-cloud-os_login README](google-cloud-os_login/README.md)
-- [google-cloud-os_login API documentation](https://googleapis.dev/ruby/google-cloud-os_login/latest)
-- [google-cloud-os_login on RubyGems](https://rubygems.org/gems/google-cloud-os_login)
-- [Google Cloud DNS documentation](https://cloud.google.com/compute/docs/oslogin/rest/)
+This repository includes client libraries for Google Cloud Platform services,
+along with a selected set of Google services unrelated to the cloud platform.
 
-### Phishing Protection
+Most libraries here are automatically generated by the
+[GAPIC Generator](https://github.com/googleapis/gapic-generator-ruby). A small
+number are written and maintained by hand.
 
-- `gem install google-cloud-phishing_protection`
-- [google-cloud-phishing_protection README](google-cloud-phishing_protection/README.md)
-- [google-cloud-phishing_protection API documentation](https://googleapis.dev/ruby/google-cloud-phishing_protection/latest)
-- [google-cloud-phishing_protection on RubyGems](https://rubygems.org/gems/google-cloud-phishing_protection)
-- [Phishing Protection documentation](https://cloud.google.com/phishing-protection/docs/)
+## Using the client libraries
+
+These client library rubygems each include classes and methods that can be used
+to make authenticated calls to specific Google APIs. Some libraries also
+include additional convenience code implementing common client-side workflows
+or best practices.
 
-### Cloud Pub/Sub
+In general, you can expect to:
 
-- `gem install google-cloud-pubsub`
-- [google-cloud-pubsub README](google-cloud-pubsub/README.md)
-- [google-cloud-pubsub API documentation](https://googleapis.dev/ruby/google-cloud-pubsub/latest)
-- [google-cloud-pubsub on RubyGems](https://rubygems.org/gems/google-cloud-pubsub)
-- [Google Cloud Pub/Sub documentation](https://cloud.google.com/pubsub/docs)
+1.  Activate access to the API by creating a project on the Google Cloud
```

Comment:
may be we could link this to the section below

Reply:
Yeah, good call on linking to the detail sections.

Reply:
Done